src/test/java/org/jboss/ejb/client/test/OOMEInInvocationTestCase.java

```diff
@@ -0,0 +1,126 @@
+import org.jboss.ejb.client.legacy.JBossEJBProperties;
+import org.jboss.ejb.client.test.common.Echo;
+import org.jboss.logging.Logger;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * This test throws OutOfMemoryException on receiver through byteman, we check the log returned by XNIO contains
+ * all the information
+ * @author tmiyar
+ *
+ */
+@RunWith(BMUnitRunner.class)
+@BMScript(dir="src/test/resources")
+public class OOMEInInvocationTestCase extends AbstractEJBClientTestCase {
+
+    private static final Logger logger = Logger.getLogger(OOMEInInvocationTestCase.class);
+    private static final String PROPERTIES_FILE = "jboss-ejb-client.properties";
+
+    /**
+     * Do any general setup here
+     * @throws Exception
+     */
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        // trigger the static init of the correct proeprties file - this also depends on running in forkMode=always
+        JBossEJBProperties ejbProperties = JBossEJBProperties.fromClassPath(OOMEInInvocationTestCase.class.getClassLoader(), PROPERTIES_FILE);
+        JBossEJBProperties.getContextManager().setGlobalDefault(ejbProperties);
+
+        // Launch callback if needed
+        ClassCallback.beforeClassCallback();
+    }
+
+    /**
+     * Do any test specific setup here
+     */
+    @Before
+    public void beforeTest() throws Exception {
+        // start server
+            startServer(0, 6999);
+            // deploy a stateful bean
+            deployStateless(0);
+            System.clearProperty("echo");
+     }
+
+    /**
+     * Test SLSB invocation
+     *
+     * scenario:
+     *   invoked bean available on node1
+     *   strong affinity is set to Affinity.NONE
+     * expected result:
+     *   invocation will fail and a message containing the invoked method will be displayed
+     */
+    @Test
+    public void testSLSBInvocation() {
+        
+        Assert.assertEquals("echo system property exists", System.getProperty("echo"), null);
+        
+        Affinity expectedStrongAffinity = Affinity.NONE;
+        
+        // create a proxy for SLSB
+        final StatelessEJBLocator<Echo> statelessEJBLocator = StatelessEJBLocator.create(Echo.class, STATELESS_IDENTIFIER, expectedStrongAffinity);
+        Echo proxy = EJBClient.createProxy(statelessEJBLocator);
+        Assert.assertNotNull("Received a null proxy", proxy);
+
+        // invoke on the proxy
+        logger.info("Invoking on proxy...");
+        final String message = "hello!";
+        try {
+            proxy.echo(message);
+        } catch (RuntimeException e) {
+            //don't do anything, it is expected
+        }
+        // check the property contents
+        Assert.assertEquals("method echo not in error message", System.getProperty("echo"), "true");
+    }
```

Comment:
It seems the order of the params (expected, actual) are inverted in uses of `assertEquals` method.